trust/README.md

```diff
@@ -1,198 +1,437 @@
+
+1.  Pull the Alpine image with the `sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7` digest.
+
+   ```
+   $ sudo docker pull alpine@sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7
+
+   sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7: Pulling from library/alpine
+   6589332ef57c: Pull complete
+   Digest: sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7
+   Status: Downloaded newer image for alpine@sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7
+   ```
+
+2. Check that the pull succeeded.
+
+   ```
+   $ sudo docker images --digests alpine
+
+   REPOSITORY          TAG                 DIGEST                                                                    IMAGE ID            CREATED             SIZE
+   alpine              edge                <none>                                                                    3fc33d6d5e74        4 weeks ago         4.799 MB
+   alpine              <none>              sha256:b7233dafbed64e3738630b69382a8b231726aa1014ccaabc1947c5308a8910a7   79c898d40088        7 weeks ago         4.799 MB
+   ```
+
+   Notice that there are now two Alpine images in the Docker Hosts local repository. One lists the `edge` tag. The other lists `<none>` as the tag along with the `b7233daf...` digest.
+
+The content addressable storage model used by Docker images means that any changes made to an image will result in a new digest for the updated image. This means it is not possible for two images with different contents to have the same digest.
+
+In this step you learned how to pull images by digest and list image digests using the `docker images` command. For more information about the `docker pull` command, see the [documentation](https://docs.docker.com/engine/reference/commandline/pull/).
+
+# <a name="trust"></a>Step 3: Docker Content Trust
+
+It's not easy to find the digest of a particular image tag. This is because it is computed from the hash of the image contents and stored in the image manifest. The image manifest is then stored in the Registry. This is why we needed a `docker pull` by tag to find digests previously. It would also be desirable to have additional security guarantees such as image freshness.
+
+Enter Docker Content Trust: a system currently in the Docker Engine that verifies the publisher of images without sacrificing usability. Docker Content Trust implements [The Update Framework](https://theupdateframework.github.io/) (TUF), an NSF-funded research project succeeding Thandy of the Tor project. TUF uses a key hierarchy to ensure recoverable key compromise and robust freshness guarantees.
+
+> **NOTE TO LAB OWNER: I'M UNCERTAIN ABOUT THE STATEMENT "SUCCEEDING THANDY OF THE TOR PROJECT". I'M NOT DOUBTING IT'S CORRECT, I JUST KNOW NOTHING ABOUT THIS SO AM NOT SURE IF I'VE MISUNDERSTOOD IT....**
```

Comment:
thanks for flagging, it's correct 👍  

https://github.com/theupdateframework/tuf/blob/master/LICENSE.txt#L32